packages/camera/camera_android_camerax/example/lib/camera_controller.dart

```diff
@@ -0,0 +1,554 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
```

Comment:
This is an exact copy from [`camera_android`](https://github.com/flutter/plugins/blob/main/packages/camera/camera_android/example/lib/camera_controller.dart).